### DISCUSSION TITLE
6x12 bitmap png font

### DISCUSSION THREAD

**Discussion by defdefred** [VendelinSlezak/BleskOS#66] [Show and tell]
0xxxxxxx are standard ASCII (128).
10xxxxx0 are upper combining diacritics (32).
10xxxxx1 are lower combining diacritics (32).
11xxxxxx are other characters (64).
work on going...
![Modele Font 12x6](https://github.com/VendelinSlezak/BleskOS/assets/17281367/d0443d29-fd30-4526-b940-84d9f72d032e)

**Comment by VendelinSlezak** [comment]
It looks nice. I am looking forward to this :)

**Comment by defdefred** [comment]
I'm looking for for an optimized 8 bits representation of all Latin possible glyphs using combinations and easy matching to characters type with binary operation.
I'm also looking for a way to use the first useless 32 control characters (000xxxxx) as formatting tags (markdown like).
I will give unbelievable speed for document rendering...
